pocs-capstone/frontend/src/components/AnimateChoice.js

```diff
@@ -35,15 +41,15 @@ const AnimateChoice = () => {
     };
     // handle selection
```

Comment:
add a new handler called handleSelect() to match up with the Carousel component onSelect handler: 
```    
const handleSelect = (selectedPet) => {
        setSelectedIndex(selectedPet)
}
```